setup.py

```diff
@@ -42,12 +50,7 @@
         'aiodocker',
     ],
     python_requires='>=3.5',
-    install_requires=[
-        'setuptools==36.2.0',
-        'aiohttp==2.2.3',
-        'yarl>=0.10',
-    ],
-    extras_require={
-        'test': ['pytest', 'pytest-asyncio', 'flake8'],
-    }
+    install_requires=requirements,
```

Comment:
I suspect the library should not pin own requirements -- otherwise we need publish new aiodocker on every yarl/aiohttp update.